parsing/schools/jhu/courses.py

```diff
@@ -112,18 +112,29 @@ def _compute_size_enrollment(self, course):
             waitlist = None
         return (section_size, section_enrolment, waitlist)
 
-    def _load_ingestor(self, course, section):
+    def _load_ingestor(self, school, course, section):
+        self.ingestor['sub_school'] = school;
         section_details = section[0]['SectionDetails']
         try:
             num_credits = float(course['Credits'])
         except:
             num_credits = 0
 
-        # Load core course fields
-        self.ingestor['areas'] = filter(lambda a: a != "None",
-                                        course['Areas'].split(','))
-        if course['IsWritingIntensive'] == "Yes":
-            self.ingestor['areas'] += ['Writing Intensive']
+
+        areas = []
+        if course['Areas'][0] != 'B' and course['Areas'][0] != 'P' and course['Areas'][0] != 'A' and course['Areas'] != 'None':
```

Comment:
make course['Areas'][0] a variable; add a note about what B,P,A are - even better, make this a function called checkHomewood so its explicit and also reusable